### ISSUE TITLE
Warning: <dependency> depends on <dependency>. CommonJS or AMD dependencies can cause optimization bailouts.

### ISSUE THREAD

**Issue by SimonSiefke** [okta/samples-js-angular#167]
![warnings](https://github.com/okta/samples-js-angular/assets/23744935/3b93f52d-3d1d-4e2d-9482-e317a4931a13)

### Expected behaviour
When running the custom login page example, no warnings are logged to the console

### Actual behaviour
Lots of warnings are logged to the node console, the warnings all have the form `<dependency> depends on <dependency>. CommonJS or AMD dependencies can cause optimization bailouts.`

### Output

```
Warning: /home/simon/.cache/repos/samples-js-angular/custom-login/node_modules/@giomamaladze/ng2-semantic-ui/__ivy_ngcc__/fesm2015/giomamaladze-ng2-semantic-ui.js depends on 'bowser'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/simon/.cache/repos/samples-js-angular/custom-login/node_modules/@giomamaladze/ng2-semantic-ui/__ivy_ngcc__/fesm2015/giomamaladze-ng2-semantic-ui.js depends on 'date-fns/locale/en-US'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/simon/.cache/repos/samples-js-angular/custom-login/node_modules/@giomamaladze/ng2-semantic-ui/__ivy_ngcc__/fesm2015/giomamaladze-ng2-semantic-ui.js depends on 'extend'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/simon/.cache/repos/samples-js-angular/custom-login/node_modules/@giomamaladze/ng2-semantic-ui/__ivy_ngcc__/fesm2015/giomamaladze-ng2-semantic-ui.js depends on 'is-ua-webview'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/simon/.cache/repos/samples-js-angular/custom-login/node_modules/@giomamaladze/ng2-semantic-ui/__ivy_ngcc__/fesm2015/giomamaladze-ng2-semantic-ui.js depends on 'ts-input-mask'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/simon/.cache/repos/samples-js-angular/custom-login/node_modules/@okta/okta-auth-js/esm/browser/core/AuthStateManager.js depends on 'p-cancelable'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/simon/.cache/repos/samples-js-angular/custom-login/node_modules/@okta/okta-auth-js/esm/browser/myaccount/types.js depends on 'cross-fetch'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/simon/.cache/repos/samples-js-angular/custom-login/node_modules/@okta/okta-auth-js/esm/browser/myaccount/types.js depends on 'tiny-emitter'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/simon/.cache/repos/samples-js-angular/custom-login/node_modules/@okta/okta-signin-widget/dist/esm/packages/@okta/courage-dist/esm/src/courage/util/Clipboard.js depends on 'clipboard'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/simon/.cache/repos/samples-js-angular/custom-login/node_modules/@okta/okta-signin-widget/dist/esm/packages/@okta/courage-dist/esm/src/courage/util/jquery-wrapper.js depends on 'jquery'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/simon/.cache/repos/samples-js-angular/custom-login/node_modules/@okta/okta-signin-widget/dist/esm/src/util/TimeUtil.js depends on '@sindresorhus/to-milliseconds'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/simon/.cache/repos/samples-js-angular/custom-login/node_modules/@okta/okta-signin-widget/dist/esm/src/util/TimeUtil.js depends on 'parse-ms'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/simon/.cache/repos/samples-js-angular/custom-login/node_modules/@okta/okta-signin-widget/dist/esm/src/v1/controllers/EnrollDuoController.js depends on 'q'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

Warning: /home/simon/.cache/repos/samples-js-angular/custom-login/node_modules/unload/dist/es/index.js depends on 'detect-node'. CommonJS or AMD dependencies can cause optimization bailouts.
For more info see: https://angular.io/guide/build#configuring-commonjs-dependencies

```

**Comment by denysoblohin-okta** [comment]
Not all dependencies of okta-auth-js and okta-signin-widget yet support ESM.
To suppress warnings, `allowedCommonJsDependencies` [can be used](https://angular.io/guide/build#configuring-commonjs-dependencies)
Internal ref: OKTA-628076

**Comment by jaredperreault-okta** [comment]
Resolved in #168
